src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -269,6 +269,10 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
     }
   );
 
+  const parentSpace = ownedVenues.find(
```

Comment:
Same consideration like above: if possible to reduce O(n) down to O(1) without wasting too much time on development, maybe it should be done.

Reply:
done 6b9f914